scripts/bootstrap-aio.yml

```diff
@@ -37,7 +37,7 @@
   include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
   vars:
     bootstrap_host_apt_distribution_suffix_list: "{{ (lookup('env', 'RPCO_APT_ARTIFACTS_AVAILABLE') | bool) | ternary([], ['updates', 'backports']) }}"
-    scenario: "{% if lookup('env', 'DEPLOY_MAGNUM') == 'yes' %}magnum{% else %}{{ lookup('env', 'DEPLOY_CEPH') |bool |ternary('ceph','swift') }}{%endif%}"
+    bootstrap_host_scenario: "{{ (lookup('env', 'DEPLOY_CEPH') | bool) | ternary('ceph','swift') }}"
```

Comment:
Should this be `scenario` or `bootstrap_host_scenario`?

Reply:
Ah, you're right - I got it mixed up. In Newton it's scenario, in Ocata onwards it's bootstrap_host_scenario. Thanks!